cli-main.ts

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
```

Comment:
Since this filename was used previously to mean something else (it was the main js file for just the EWCLI) it's a little confusing.  Maybe this should be index.js or something similar so its clear that this new file has a new purpose vs. what cli-main.js meant before.

Reply:
In my mind, cli-main is the main file for both edgeworkers and edgeKV and main class for package.json. If you think its confusing I can rename it to "index"

Reply:
It could just be me that's confused since I know too much of the code details :-)  Beyond me, the only other confusion is that cli-main.ts was a filename used in previous EWCLI versions for a completely different purpose than how you are using it now in 0.5.0.  The name doesn't have to be index.js if you have another name in mind that's fine.